fun.py

```diff
@@ -195,9 +195,10 @@ def pizza(self):
             yield (pixels, 0.1, False)
 
     def fade(self):
-      bright = 255
-      pixels = [(bright,bright,bright)] * numLEDs
-      while bright > 0:
-        bright -= 1
-        pixels = [(bright,bright,bright)] * numLEDs
-        self.client.put_pixels(pixels)
+        whiile True:
+            bright = 255
+            pixels = [(bright,bright,bright)] * numLEDs
+            while bright > 0:
+                bright -= 1
+            pixels = [(bright,bright,bright)] * numLEDs
+            yield (pixels, 3, False)
```

Comment:
3 seconds would be a long time between frames.  255 * 3 is almost a 10 minute fade